debian/rules

```diff
@@ -1,114 +1,25 @@
 #!/usr/bin/make -f
-# -*- makefile -*-
-# Sample debian/rules that uses debhelper.
-# GNU copyright 1997 to 1999 by Joey Hess.
 
-# Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
-CFLAGS = -Wall -Wno-format -g
+export DEB_CFLAGS_MAINT_APPEND  = -Wall -Wno-format -Wno-format-security -g -DTAP_AND_HOLD
+export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
 
-ifneq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
-	CFLAGS += -O0
-else
-	CFLAGS += -O2
-endif
-ifeq (,$(findstring nostrip,$(DEB_BUILD_OPTIONS)))
-	INSTALL_PROGRAM += -s
-endif
-ifneq (,$(findstring thumb,$(DEB_BUILD_OPTIONS)))
-        CFLAGS += -mthumb
-endif
 ifeq (,$(findstring nolauncher,$(DEB_BUILD_OPTIONS)))
 	CONFIGURE_OPTIONS += --enable-maemo-launcher
 	USE_MAEMO_LAUNCHER = true
 endif
 
-CFLAGS += -DTAP_AND_HOLD
```

Comment:
Are you sure this shall be removed?

Reply:
He moved it to CFLAGS_MAINT.
But I agree, it's better to leave it around for better observing what has changed.

Reply:
Moved to CPPFLAGS_MAINT in 780bdf22fd425ba5acfd175baf70609250d7907c